data/core/units/elves/Lady.cfg

```diff
@@ -17,7 +17,7 @@
     {AMLA_DEFAULT}
     cost=53
     usage=null
-    description= _ "Usually strongly connected with the earthly aspect of Elven magic, noble ladies of the elvish enclaves are associated with wisdom and preservation. As those entrusted with maintaining many of the more venerated Elvish traditions, they play a large role in safeguarding the tranquility and natural wonder of their renowned forest sanctuaries. Unlike their counterparts in the lords, many of whom gain great individual insight through their use of arcane magic, most ladies spread their power throughout the societal orders that they guide, often at some cost to their own personal abilities. For this reason, they serve most often as diplomats and peacekeepers, holding high authority during times of amity and taking a lesser role in times of war."
+    description= _ "Usually strongly connected with the earthly aspect of Elven magic, noble Ladies of the elvish enclaves are associated with wisdom and preservation. As those entrusted with maintaining many of the more venerated Elvish traditions, they play a large role in safeguarding the tranquility and natural wonder of their renowned forest sanctuaries. Unlike their counterparts in the lords, many of whom gain great individual insight through their use of arcane magic, most Ladies spread their power throughout the societal orders that they guide, often at some cost to their own personal abilities. For this reason, they serve most often as diplomats and peacekeepers, holding high authority during times of amity and taking a lesser role in times of war."
```

Comment:
Interesting: 'Elven' and 'Elvish' should probably be lower-case. Conversely, 'lords' should be capitalised (as counterpart to 'Ladies').